docs/developers-guide/sink-targets/substreams-sink-clickhouse.md

```diff
@@ -0,0 +1,93 @@
+
+
+### Quickstart
+
+1. Install `substreams-sink-clickhouse` by using the pre-built binary release [available in the releases page](https://github.com/Aleno1/substreams-sink-clickhouse/releases). Extract `substreams-sink-clickhouse` binary into a folder and ensure this folder is referenced globally via your `PATH` environment variable.
+
+    > **Note** Or install from source directly `go install github.com/Aleno1/substreams-sink-clickhouse/cmd/substreams-sink-clickhouse@latest`.
+
+1. Start Docker Compose:
+
+    ```bash
+    docker compose up
+    ```
+
+    > **Note** Feel free to skip this step if you already have a running ClickHouse instance accessible, don't forget to update the connection string in the command below.
+
+2. Setup ClickHouse
+
+    Connect to ClickHouse
+
+    ```bash
+    docker compose exec ch_server clickhouse-client -u dev-node --password insecure-change-me-in-prod -h localhost
+    ```
+
+    And create necessary tables to run the sink
+
+    ```sql
+        CREATE TABLE block_meta
+    (
+        id          String, 
+        at          String,
+        number      Int32,
+        hash        String,
+        parent_hash String,
+        timestamp   String,
+        PRIMARY KEY (id),
+    )
+    ENGINE = MergeTree()
+    ORDER BY id;
+
+    CREATE TABLE cursors
+    (
+        id         String,
+        cursor     String,
+        block_num  Int64,
+        block_id   String,
+        PRIMARY KEY (id)
+    ) ENGINE = MergeTree()
+    ORDER BY id;
+    ```
+
+    > **Note** Each create table query must be run independently as ClickHouse doesn't support multiple create table queries at once.
+
+3. Run the sink
+
+    Use the precompiled Ethereum Block Meta [substreams](https://github.com/streamingfast/substreams-eth-block-meta/releases/download/v0.4.1/substreams-eth-block-meta-v0.4.1.spkg)
+
+    > **Note** To connect to Substreams you will need an authentication token, follow this [guide](https://substreams.streamingfast.io/reference-and-specs/authentication) to obtain one.
+
+    ```shell
+    substreams-sink-clickhouse run \
+        "clickhouse://dev-node:insecure-change-me-in-prod@localhost:8123" \
+        "mainnet.eth.streamingfast.io:443" \
+        https://github.com/streamingfast/substreams-eth-block-meta/releases/download/v0.4.1/substreams-eth-block-meta-v0.4.1.spkg \
+        db_out
+    ```
+
+### Output Module
+
+To be accepted by `substreams-sink-clickhouse`, your module output's type must be a [sf.substreams.sink.database.v1.DatabaseChanges](https://github.com/streamingfast/substreams-database-change/blob/develop/proto/substreams/sink/database/v1/database.proto#L7) message. The Rust crate [substreams-data-change](https://github.com/streamingfast/substreams-database-change) contains bindings and helpers to implement it easily. Some project implementing `db_out` module for reference:
```

Comment:
The repo changed for `substreams-sink-database-changes` (but the Rust crate name is still `substreams-database-change` to avoid breaking changes for now)
```suggestion
To be accepted by `substreams-sink-clickhouse`, your module output's type must be a [sf.substreams.sink.database.v1.DatabaseChanges](https://github.com/streamingfast/substreams-sink-database-changes/blob/develop/proto/sf/substreams/sink/database/v1/database.proto#L7) message. The Rust crate [substreams-database-change](https://github.com/streamingfast/substreams-sink-database-changes) contains bindings and helpers to implement it easily. Some project implementing `db_out` module for reference:
```